idle-batch/src/main/kotlin/com/swm/idle/batch/crawler/WorknetPostCrawler.kt

```
@@ -0,0 +1,142 @@
+
+        val crawledPostings = mutableListOf<CrawledJobPostingDto>()
+        repeat(end) { i ->
+            val originalWindow = driver.windowHandle
+            val titleElement = findElementSafe(By.xpath("//*[@id=\"list${i+1}\"]/td[1]/div/div[2]/a")) ?:  return@repeat
+
+            moveToPostDetailWindow(titleElement, originalWindow)
+
+            try {
+                val post: CrawledJobPostingDto = createPost()
+                crawledPostings.add(post)
+            } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)